scripts/app_load_params_check.py

```diff
@@ -2,42 +2,10 @@
 
 from argparse import ArgumentParser
 from pathlib import Path
-from typing import Dict
-from collections import defaultdict
 import json
 
-APP_LOAD_PARAMS_ALLOWED = {
-    "targetId",
-    "targetVersion",
-    "apiLevel",
-    "fileName",
-    "icon",
-    "curve",
-    "path",
-    "path_slip21",
-    "appName",
-    # "signature", # Reserved for internal usage
-    # "signApp", # Reserved for internal usage
-    "appFlags",
-    # "bootAddr", # Deprecated?
-    # "rootPrivateKey", # Should not be used for app deployment
-    # "signPrivateKey", # Should not be used for app deployment
-    # "apdu", # Should not be used for app deployment
-    # "deployLegacy", # Deprecated?
-    "delete",
-    # "params", # Deprecated?
-    "tlv",
-    "dataSize",
-    "appVersion",
-    # "offline", # Should not be used for app deployment
-    # "offlineText", # Should not be used for app deployment
-    # "installparamsSize", # Deprecated?
-    "tlvraw",
-    # "dep", # Deprecated?
-    "nocrc",
-}
 
-APP_LOAD_PARAMS_VALUE_CHECK = {
```

Comment:
This will break the compatibility with `scripts/app_load_params_gen_db.py`

Reply:
Fixed